core/templates/dev/head/appSpec.js

```diff
@@ -224,4 +224,30 @@ describe('Code Normalization', function() {
       'cde'
     );
   });
+
+  it('should normalize multiple spaces within a line', function() {
+    expect(cns.getNormalizedCode(
+      'abcdefg\n' +
+      '    hij    klm\n' +
+      '    ab "cde fgh"\n'
+    )).toBe(
+      'abcdefg\n' +
+      '    hij klm\n' +
+      '    ab "cde fgh"'
+    );
+  });
+
+  it('should be case-insensitive within a string but not outside', function() {
+    expect(cns.getNormalizedCode(
+      'abcdefg\n' +
+      '    ABcDe \'HiJkL\'\n' +
+      '    "HiJkL    MNopq  " ABcDe\n' +
+      '    "AbC@\\"DeF7"  GhI# \'JkL-\\\'MnO3\''
+    )).toBe(
+      'abcdefg\n' +
+      '    ABcDe \'hijkl\'\n' +
+      '    "hijkl mnopq " ABcDe\n' +
+      '    "abc@\\"def7" GhI# \'jkl-\\\'mno3\''
```

Comment:
I don't think so? The string is made up of the following characters: a, b, c, @, backslash, quotation mark, d, e, f, 7.